script/cibuild

```diff
@@ -2,28 +2,19 @@
 # Usage: script/cibuild [--no-package]
 set -e
 
-# GHE appliance versions to run tests against. Remote metadata files are put in
-# place with this version at the beginning of each test and many commands have
-# conditional logic based on the remote version. Running the suite against
-# different major versions ensures we're covering these conditional paths.
-REMOTE_VERSIONS="2.19.0 2.21.0"
-
 # Enable verbose logging of ssh commands
 export GHE_VERBOSE_SSH=true
 
-# Run over all remote versions and run entire test suite against each
 res=true
-for version in $REMOTE_VERSIONS
-do
-  echo "==> Running testsuite with GHE_TEST_REMOTE_VERSION=$version"
-  export GHE_TEST_REMOTE_VERSION="$version"
-  if ! find test -name "test-*.sh" -print0 | xargs -0 -P 4 -n 1 /bin/bash; then
-    res=false
-  fi
-  echo
-done
+version=2.21.0
+echo "==> Running test suite with GHE_TEST_REMOTE_VERSION=$version"
+export GHE_TEST_REMOTE_VERSION="$version"
+if ! find test -name "test-*.sh" -print0 | xargs -0 -P 4 -n 1 /bin/bash; then
+  res=false
+fi
+echo
```

Comment:
This is introducing another place we'll need to keep updating the version with each release. It's also unnecessary as the version will always be overwriting the the same default version defined at https://github.com/github/backup-utils/blob/b774394b879d55b80a3f8d00a02454fe0bb36487/test/testlib.sh#L43-L46

We can 🔥 this entire section.

```suggestion
```

Reply:
Oh! Ok, great!

Reply:
I initially committed this suggestion, but then came to the conclusion following what other code might be deleted in this thread https://github.com/github/backup-utils/pull/615#discussion_r444481495 that we need to run the tests here at least once, so I removed the bits that were unnecessary but left the test execution.

Let me know what you think.

Reply:
🤦 oops. Yes, we still need that line.

Reply:
Cool, thanks for clarifying that for me!